controllers/workshop_controller.go

```diff
@@ -260,3 +248,32 @@ func (r *WorkshopReconciler) handleDelete(ctx context.Context, req ctrl.Request,
 
 	return ctrl.Result{}, nil
 }
+
+func (r *WorkshopReconciler) getVariables(workshop *workshopv1.Workshop, ctx context.Context) (int, string, string, ctrl.Result, error) {
```

Comment:
I don't see any benefit of it

Reply:
these variables are defined only once; so, defining method for it doesn't give any value